### PR TITLE
Fix #547: don't use select * in queries

### LINKED ISSUE #547: CosmosDB Storage is using "Select *" and should be explicit
The current implemention of CosmosDB uses the following query in two places:
```
{
        query: 'SELECT * FROM root r WHERE r.id=@id',
        parameters: [{ name: '@id', value: collectionId }]
};
```
This should be changed to use explicit fields, as using "Select *" is generally not considered best practice for writing SQL statements.

### PR DESCRIPTION
**Pull request by JuanAr**
- Changed `SELECT * FROM root r WHERE r.id=@id` into `SELECT r._self FROM root r WHERE r.id=@id`

### CODE REVIEW

File: libraries/botbuilder-azure/src/cosmosDbStorage.ts [hunk 1]
```diff
@@ -261,7 +261,7 @@ function getOrCreateCollection(client: DocumentClient, databaseLink: string, col
             value: string;
         }[];
     } = {
-        query: 'SELECT * FROM root r WHERE r.id=@id',
+        query: 'SELECT r._self FROM root r WHERE r.id=@id',
```
Comment (cleemullins) [inline libraries/botbuilder-azure/src/cosmosDbStorage.ts]: Line 235 in "getOrCreateDatabase" needs to be fixed as well.

File: libraries/botbuilder-azure/src/cosmosDbStorage.ts [hunk 2]
```diff
@@ -261,7 +261,7 @@ function getOrCreateCollection(client: DocumentClient, databaseLink: string, col
             value: string;
         }[];
     } = {
```
Comment (cleemullins) [inline libraries/botbuilder-azure/src/cosmosDbStorage.ts]: The SQL Query here "Select r._self" is quite different from the [C# Query](https://github.com/Microsoft/botbuilder-dotnet/blob/34c957e998c03b37585dc993b48148e994134c9c/libraries/Microsoft.Bot.Builder.Azure/CosmosDbStorage.cs#L143). 

This seems like an area where the 2 platforms (and soon 3, with Java coming soon) should be *exactly* aligned. 

`
QueryText = $"SELECT c.id, c.realId, c.document, c._etag FROM c WHERE c.id in ({parameterSequence})"
`

File: libraries/botbuilder-azure/src/cosmosDbStorage.ts (hunk 1 above)
Comment (JuanAr) [reply libraries/botbuilder-azure/src/cosmosDbStorage.ts]: This is done!

File: libraries/botbuilder-azure/src/cosmosDbStorage.ts (hunk 2 above)
Comment (JuanAr) [reply libraries/botbuilder-azure/src/cosmosDbStorage.ts]: This query is used for something different than checking the database existence.
The [equivalent query in JS](https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-azure/src/cosmosDbStorage.ts#L115) is exactly the same:
```
query: `SELECT c.id, c.realId, c.document, c._etag FROM c WHERE c.id in (${parameterSequence})`,
```

**Pull request merged**